src/ORM/Marshaller.php

```diff
@@ -105,6 +105,22 @@ public function one(array $data, array $options = [])
         $propertyMap = $this->_buildPropertyMap($options);
 
         $schema = $this->_table->schema();
+        $primaryKey = $schema->primaryKey();
+
+        if (array_intersect($primaryKey, array_keys($data)) == $primaryKey) {
+            $tableName = $this->_table->table();
+            $record = $this->_table->find('all');
+            foreach ($primaryKey as $pkey) {
+                $record->where(["$tableName.$pkey" => $data[$pkey] ]);
```

Comment:
What surprised me is that phpcs tests made locally didn't pointed this. I just fixed this.
Thank you